README.md

```diff
@@ -32,7 +32,7 @@ require 'prawn/emoji'
 
 Prawn::Document.generate 'foo.pdf' do
   font 'DejaVuSans.ttf'
-  text '🐟 + 🔪 = 🍣'
+  text '🐟 / 🔪 + 🍚 = 🍣'
```

Comment:
I think sushi requires sumeshi (vinegar-flavors rice)

```suggestion
  text '🐟 / 🔪 + 🍚 / 🍾) = 🍣'
```

Reply:
@exoego thank you and I fixed!
https://github.com/hidakatsuya/prawn-emoji/pull/22/commits/e299beae332b6f18e1a65ccd03c7dd0b7c3e7f56